src/mgmt/srv_container.c

```diff
@@ -53,7 +58,7 @@ ds_mgmt_cont_set_owner(uuid_t pool_uuid, d_rank_list_t *svc_ranks,
 	if (prop == NULL)
```

Comment:
nit: wouldn't it be more consistent to test for (!prop) here given you've changed (group != NULL) to (group)?

Reply:
I created `user_set` and `grp_set` as booleans so I didn't have to do multiple checks, not just checking pointers directly anymore. In the style guide I think we're still preferring the more explicit `ptr != NULL` comparison.